api_specifications/case-api-0.1.0.yaml

```diff
@@ -335,7 +334,7 @@ components:
           type: string
         stage:
           type: string
-        substage:
+        subStage:
```

Comment:
@atulgupta2024 can we add descriptions for stage/substage etc.. as well?

Reply:
done.